pyccel/codegen/printing/ccode.py

```diff
@@ -285,14 +285,6 @@ def _print_If(self, expr):
             lines.append("%s\n}" % var)
         return "\n".join(lines)
 
-    def _print_IfTernaryOperator(self, expr):
-        condition, if_exp   = expr.args[0]
-        _, else_exp         = expr.args[1]
-        condition           = self._print(condition)
-        if_exp              = self._print(if_exp)[:-1]
-        else_exp            = self._print(else_exp)[:-1]
-        return "({}) ? ({}) : ({});".format(condition, if_exp, else_exp)
```

Comment:
Is this code in a branch somewhere? If not then it can be copy/pasted into the issue so it doesn't need to be written twice

Reply:
the `IfTernaryOperator` will not inherit anymore from the `if` class. so I think this code will not be useful.

Reply:
Ah good point :+1: